packages/lib-classifier/src/store/SubjectStore.js

```diff
@@ -111,6 +111,17 @@ const SubjectStore = types
       }
     }
 
+    function append (newSubjects) {
+      newSubjects.forEach(subject => {
```

Comment:
This fails if I change it to
```js
newSubjects.forEach(self.resources.put)
```
and I can't see why. `self.resources.merge(newSubjects)` also throws errors, so I've left the loop in.

Reply:
`merge` expects an array of entries, so `[['1', { location: 'foo.jpg' }], ['2', { location: 'bar.jpg' }]]` or an object. I'm not sure why calling put directly would fail either.

Reply:
Maybe `self.resources` isn't a map at runtime, so the `put` method doesn't exist yet? I'm not sure either.

Reply:
I had another look at this and `newSubjects.forEach(self.resources.put)` fails because `self.resources` isn't a valid MST node.